app/controllers/api/v1/worksheets_controller.rb

```diff
@@ -18,7 +18,7 @@ def edit
 
   def update
     image_url = remote_image_url
-    prepped_params = worksheet_params.slice(:image_url, :photo).merge({ image_url: image_url })
+    prepped_params = worksheet_params.except(:image_url, :photo).merge({ image_url: image_url })
```

Comment:
So prepped_params is an array that is returned from the query which looks for worksheet_params with the condition that image_url is equal to remote_image_url?

Reply:
```
[1] pry(#<Api::V1::WorksheetsController>)> prepped_params.class
=> ActionController::Parameters
```
prepped_params isn't an array, it's an ActionController class. 
It calls the private method `worksheet_params` https://github.com/odaiba-app/server/pull/80/files#diff-e626411ac3c7c3e2593ab9895451461cR60 which is a selection of the permitted params (so-called 'strong params' - it's there so that users with malicious intent can't pass parameters that could break our code).
In line 20 we are uploading the image from the params to cloudinary and returning the cloudinary url, so since we want to save that url and not something else, we are removing the `:image_url, :photo` params and merging the new image url